docs/python-api/itwinai.loggers.rst

```diff
@@ -1,9 +1,8 @@
 itwinai.loggers
-===============
+-----------------
```

Comment:
I have not checked, but are you sure that this is not going to mess with the headings hierarchy ?

Reply:
Changed it :)